src/script/components/TitleBar/TitleBar.tsx

```diff
@@ -251,9 +251,9 @@ export const TitleBar: React.FC<TitleBarProps> = ({
               />
             )}
 
-            <h2 className="conversation-title-bar-name-label" data-uie-name="status-conversation-title-bar-label">
+            <span className="conversation-title-bar-name-label" data-uie-name="status-conversation-title-bar-label">
               {displayName}
-            </h2>
+            </span>
```

Comment:
Shouldn't this be an H3? (I'm looking at the Addesso chat and don't see why this would be a span)

If for some reason it shouldn't be a heading, it would probably be a `<p>`tag rather than a `<span>` no?

Reply:
oh, it's interact-able...